modules/nux/modules.php

```diff
@@ -130,36 +141,34 @@ public function process() {
                     $this->session->record_unsaved('IMAP server added');
                     $this->session->secure_cookie($this->request, 'hm_reload_folders', '1');
                     $this->session->close_early();
-                }
-                else {
+                } else {
                     Hm_Msgs::add('ERRAn Error Occurred');
                 }
-            }
-            elseif (array_key_exists('error', $this->request->get)) {
-                Hm_Msgs::add('ERR'.ucwords(str_replace('_', ' ', $this->request->get['error'])));
-            }
-            else {
+            } elseif (array_key_exists('error', $this->request->get)) {
+                Hm_Msgs::add('ERR' . ucwords(str_replace('_', ' ', $this->request->get['error'])));
+            } else {
                 Hm_Msgs::add('ERRAn Error Occurred');
             }
             $this->save_hm_msgs();
-            Hm_Dispatch::page_redirect('?page=servers');
```

Comment:
This line is too important to redirect to servers page. This handler runs on home page